ui/src/app/model/enums/publisherRole.ts

```diff
@@ -1,8 +1,8 @@
 export enum PublisherRole {
-    CA  = "CA",
-    AG = "AG",
-    VSL = "VSL",
-    ATH = "ATH",
-    PLT = "PLT",
-    TR = "TR"
-  }
\ No newline at end of file
+  CA = "CA",    // Carrier
+  AG = "AG",    // ???
+  VSL = "VSL",  // Vessel?
+  ATH = "ATH",  // Port Authority
+  PLT = "PLT",  // Pilot
+  TR = "TR"     // Terminal
```

Comment:
According to the Bussiness Requirements, there are the following 6 roles:


Carrier:
o	OPS for Carrier Operations  (I assume this is what we know as `CA`)
o	AGE for Carrier local Agent (I assume this is what we know as `AG`)
o	VSL for Vessel
Port:
o	ATH for Port Authorities
o	PLT for Port Pilot
Terminal: 
o	TRM for Terminal (I assume we use `TR` here)

Reply:
(so we can replace the `?`s with better comments)

Reply:
@nt-as done!